docs/overview/high-level-summary/index.md

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
Renamed from "Bird's Eye View" because I think this title is a bit more common and less colloquial, and easier to translate.